### PR TITLE
Use chart "locale" if it is not specified in the adatpter options

### LINKED ISSUE #71: Use chart "locale" if it is not specified in the adatpter options
When the following PR https://github.com/chartjs/Chart.js/pull/10528 in Chart.js project will be released, the adapter should be updated in order to use the chart locale options if there isn't any locale definition in the adapter options.

### PR DESCRIPTION
**Pull request by stockiNail**
Fix #71 
Answer to comment https://github.com/chartjs/chartjs-adapter-luxon/issues/46#issuecomment-983796300

## Missing

- [x] Release new CHART.JS version
- [x] Add test cases using chart locale.
- [x] Add doc in the README

### CODE REVIEW

**Comment by LeeLenaleee** [comment]
@stockiNail chart.js v3.9 has been released with the necessary changes

**Comment by stockiNail** [comment]
Thank you @LeeLenaleee. I need to finish a couple of other things for another project and then I'll take care.

**Comment by LeeLenaleee** [comment]
Yeah no problem, no hurry, just wanted to let you know :)

**Comment by stockiNail** [comment]
> Yeah no problem, no hurry, just wanted to let you know :)

Ready for review! Thank you!

File: README.md [hunk 1]
```diff
@@ -45,7 +45,7 @@ Namespace: `options.scales[scaleId].adapters.date`
 
 | Name | Type | Default | Description
 | ---- | ---- | ------- | -----------
-| `locale` | `string` | `undefined` | Set a locale to set on the resulting `DateTime` instance. By default, Luxon uses the system's locale. See [`Intl` locale](https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Global_Objects/Intl#locale_identification_and_negotiation).
+| `locale` | `string` | `undefined` | Set a locale to set on the resulting `DateTime` instance. If not set, the adapter will use the [locale](https://www.chartjs.org/docs/latest/configuration/locale.html), defined at chart options level. By default, Luxon uses the system's locale. See [`Intl` locale](https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Global_Objects/Intl#locale_identification_and_negotiation).
```
Comment (LeeLenaleee) [inline README.md]:
```suggestion
| `locale` | `string` | `undefined` | Set a locale to use on the resulting `DateTime` instance. If not set, the adapter will use the [locale](https://www.chartjs.org/docs/latest/configuration/locale.html), defined at chart options level. By default, Luxon uses the system's locale. See [`Intl` locale](https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Global_Objects/Intl#locale_identification_and_negotiation).
```

Not 100% sure but this feels like its more correct, reads a lot better to me.

For the rest lgtm

**Review by LeeLenaleee — APPROVED** [review]

**Review by kurkle — APPROVED** [review]

**Comment by kurkle** [comment]
@stockiNail can you submit a version bump?

**Comment by stockiNail** [comment]
> @stockiNail can you submit a version bump?

Yes, give some minutes

**Comment by stockiNail** [comment]
> @stockiNail can you submit a version bump?

@kurkle done!

**Pull request merged**